lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -4563,6 +4604,7 @@ class DecomposeComplexOpsPass
     addPatternIfTargetOpIsIllegal<DecomposeAtenCrossEntropyLossOp>(patterns);
     addPatternIfTargetOpIsIllegal<DecomposeAtenVarMeanDimOp>(patterns);
     addPatternIfTargetOpIsIllegal<DecomposeAtenTopkOp>(patterns);
+    addPatternIfTargetOpIsIllegal<DecomposeAtenScatterValueOp>(patterns);
```

Comment:
You also need to add this to https://github.com/llvm/torch-mlir/blob/b9b3af8003df35e7094525f9aea3ca605a0da9f9/lib/Dialect/Torch/Transforms/LowerToBackendContract.cpp#L364